pkg/apiserver/statement/queries.go

```diff
@@ -128,35 +128,20 @@ func QueryStmtTypes(db *gorm.DB) (result []string, err error) {
 // endTime: 1586845800
 // schemas: ["tpcc", "test"]
 // stmtTypes: ["select", "update"]
+// fields: ["digest_text", "sum_latency"]
 func QueryStatementsOverview(
 	db *gorm.DB,
 	beginTime, endTime int,
 	schemas, stmtTypes []string,
-	text string) (result []Model, err error) {
-	fields := getAggrFields(
-		"plan_count",
-		"table_names",
-		"schema_name",
-		"digest",
-		"digest_text",
-		"sum_latency",
-		"exec_count",
-		"max_latency",
-		"avg_latency",
-		"min_latency",
-		"avg_mem",
-		"max_mem",
-		"sum_errors",
-		"sum_warnings",
-		"avg_parse_latency",
-		"max_parse_latency",
-		"avg_compile_latency",
-		"max_compile_latency",
-		"avg_cop_process_time",
-		"max_cop_process_time")
+	text string,
+	fields []string,
+) (result []Model, err error) {
+	fields = append(fields, "schema_name", "digest", "sum_latency") // "schema_name", "digest" for group, "sum_latency" for order
+	aggrFields := getAggrFields(fields...)
```

Comment:
I'm not sure whether this will modify the original array. Anyway maybe it can be:

```go
getAggrFields("schema_name", "digest", "sum_latency", fields...)
```

Reply:
This prompts compile error as well:

![image](https://user-images.githubusercontent.com/1284531/93317111-a4384280-f83f-11ea-968b-8ce59a3a9f14.png)